src/app.module.ts

```diff
@@ -3,13 +3,26 @@ import { ConfigModule, ConfigService } from '@nestjs/config';
 import { JwtModule, JwtModuleOptions } from '@nestjs/jwt';
 import { PassportModule } from '@nestjs/passport';
 import { TypeOrmModule, TypeOrmModuleOptions } from '@nestjs/typeorm';
+import * as Joi from 'joi';
 import { LoginController } from './controllers/login.controller';
 import { User } from './entities/user.entity';
 import { LoginService } from './services/login.service';
 
 @Module({
   imports: [
-    ConfigModule.forRoot({ envFilePath: '.development.env', isGlobal: true }),
+    ConfigModule.forRoot({
+      envFilePath: '.development.env',
+      isGlobal: true,
+      validationSchema: Joi.object({
+        POSTGRES_HOST: Joi.string(),
+        POSTGRES_PORT: Joi.number().integer(),
+        POSTGRES_USER: Joi.string(),
+        POSTGRES_PASSWORD: Joi.string(),
+        POSTGRES_DB: Joi.string(),
+        JWT_SECRET: Joi.string().min(6),
```

Comment:
Validating secret is not required. I suggest validating if is a string.
```suggestion
        JWT_SECRET: Joi.string(),
```